main.js

```diff
@@ -0,0 +1,4 @@
+$(document).ready(function(){
+    $('.header').height($(window).height());
```

Comment:
Welcome to the world of jQuery! The first framework you'll learn. It's still very used and known nowadays. Hopefully, when you learn more about JS and HTML you'll start digging into other UI frameworks ;)

Reply:
Have not start learning this part yet. But I will get there next week I think.

Reply:
Yey!